aip/general/0154.md

```diff
@@ -38,7 +38,8 @@ message Book {
 ```
 
 - The etag field **must** be a string, and **must** be named `etag`.
-- The etag field **should not** be given any behavior annotations
+- The etag field on the resource **should not** be given any behavior
```

Comment:
Should the word resource be emphasized?

Reply:
I don't think so? Did you mean "bold" or "italicized" ? I don't really mind either way, but I'm not sure it is necessary given this section pertains to the resource message already. I wasn't sure about adding this phrase to begin with for the same reason, but could envision people seeing this one line and misconstruing it to apply to every etag field so I added it...